src/Security/MemberLoginForm.php

```diff
@@ -135,16 +109,69 @@ public function __construct(
 
         parent::__construct($controller, $name, $fields, $actions);
 
-        $this->setValidator(RequiredFields::create('Email', 'Password'));
+        $this->setValidator(RequiredFields::create(self::$required_fields));
```

Comment:
If this is going to be a config-overridable let's access it with the config system.